### PR TITLE
Configure docker image and web smoke tests for ADSWeb.

### PR DESCRIPTION
**Pull request by lewis-sanchez**
The changes in this PR are to set up smoke tests for ADS web in the build pipeline.

- Updates sql-web-build yaml to use web-build-image:2 instead of web-build-image:1
- Updates the Dockerfile to include wget, so that chrome can be included in the image
- Updates the sql-product-build-web yaml to use xvfb-run to allow the smoke tests to run inside of a virtual frame buffer

### CODE REVIEW

File: build/azure-pipelines/web/build/Dockerfile [hunk 1]
```diff
@@ -8,7 +8,14 @@ RUN apt-get update && apt-get upgrade -y
 
 RUN apt-get install -y libxkbfile-dev pkg-config libsecret-1-dev libxss1 dbus xvfb libgtk-3-0 \
 	libkrb5-dev git apt-transport-https ca-certificates curl gnupg-agent software-properties-common \
-	libnss3 libasound2 make gcc libx11-dev fakeroot rpm libgconf-2-4 libunwind8 g++ python
+	libnss3 libasound2 make gcc libx11-dev fakeroot rpm libgconf-2-4 libunwind8 g++ python wget
+
+# Set the Chrome repo.
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
```
Comment (kburtram) [inline build/azure-pipelines/web/build/Dockerfile:14]: not really a big deal, but from link 21 it looks like curl is already installed.  It probably could be used interchangeably with wget most the time.

File: build/azure-pipelines/web/build/Dockerfile [hunk 2]
```diff
@@ -8,7 +8,14 @@ RUN apt-get update && apt-get upgrade -y
...
+    && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list
+
+# Install Chrome.
+RUN apt-get update && apt-get -y install google-chrome-stable
```
Comment (kburtram) [inline build/azure-pipelines/web/build/Dockerfile:18]: we run `apt-get update` on line 7.  Probably reordering commands would eliminate the need to call it twice?

**Review by kburtram — APPROVED** [review]
LGTM.  Seems like the changes in the Dockerfile could be streamlined but it doesn't really matter too much.

File: build/azure-pipelines/web/build/Dockerfile (hunk 1 above)
Comment (lewis-sanchez) [reply build/azure-pipelines/web/build/Dockerfile:14]: I will look into installing with curl, so we can get wget out of there.

Comment (kburtram) [reply build/azure-pipelines/web/build/Dockerfile:14]: It's up to you.  If for some reason curl doesn't just work, I wouldn't worry about it.

**Review by kburtram — APPROVED** [review]

File: build/azure-pipelines/web/build/Dockerfile (hunk 2 above)
Comment (lewis-sanchez) [reply build/azure-pipelines/web/build/Dockerfile]: Running `apt-get install -y google-chrome-stable` without running `apt-get update` first resulted in an error. The error would say that the "google-chrome-stable" package couldn't be located, as seen here when running with curl:

image

**Pull request merged**